tests/integration/ha/helpers.py

```diff
@@ -508,15 +507,15 @@ async def wait_for_backup_system_to_settle(
                         raise Exception(f"Recovery failed for shard {shard}")
 
 
-async def delete_backup(ops_test: OpsTest, backup_id: int) -> None:
-    """Deletes a backup."""
-    # Now, check if we have finished the restore
-    unit_ip = await get_leader_unit_ip(ops_test)
-    await http_request(
-        ops_test,
-        "DELETE",
-        f"https://{unit_ip}:9200/_snapshot/{S3_REPOSITORY}/{backup_id}",
-    )
+# async def delete_backup(ops_test: OpsTest, backup_id: int, repository: str) -> None:
```

Comment:
do we no longer need this?

Reply:
It is not used anymore. The backups are deleted directly on the storage backend at the end of the tests